beetmoverscript/test/__init__.py

```diff
@@ -5,12 +5,16 @@ def get_fake_valid_config():
     return load_json(path="beetmoverscript/test/fake_config.json")
 
 
-def get_fake_valid_task():
-    return load_json(path="beetmoverscript/test/test_work_dir/task.json")
+def get_fake_valid_task(taskjson='task.json'):
+    return load_json(path="beetmoverscript/test/test_work_dir/{}".format(taskjson))
+
+
+def get_fake_balrog_props_path():
```

Comment:
What's the advantage of a function instead of a constant, in this case?

Reply:
None, really, I was attempting to keep it consistent with the way the other values were created.